morituri/common/program.py

```diff
@@ -450,58 +450,29 @@ def getTagList(self, number):
                 # htoa defaults to disc's artist
                 title = 'Hidden Track One Audio'
 
-        # here to avoid import gst eating our options
-        import gst
-
-        ret = gst.TagList()
+        tags = {}
 
-        # gst-python 0.10.15.1 does not handle unicode -> utf8 string
-        # conversion
-        # see http://bugzilla.gnome.org/show_bug.cgi?id=584445
         if self.metadata and not self.metadata.various:
-            ret["album-artist"] = albumArtist.encode('utf-8')
-        ret[gst.TAG_ARTIST] = trackArtist.encode('utf-8')
-        ret[gst.TAG_TITLE] = title.encode('utf-8')
-        ret[gst.TAG_ALBUM] = disc.encode('utf-8')
-
-        # gst-python 0.10.15.1 does not handle tags that are UINT
-        # see gst-python commit 26fa6dd184a8d6d103eaddf5f12bd7e5144413fb
-        # FIXME: no way to compare against 'master' version after 0.10.15
-        if gst.pygst_version >= (0, 10, 15):
-            ret[gst.TAG_TRACK_NUMBER] = number
+            tags['ALBUMARTIST'] = albumArtist.encode('utf-8')
+        tags['ARTIST'] = trackArtist.encode('utf-8')
+        tags['TITLE'] = title.encode('utf-8')
+        tags['DISC'] = disc.encode('utf-8')
```

Comment:
Do you mean that mutagen prefers UTF-16 instead? UTF-8 is the most common Unicode codepage.